package/pom.xml

```diff
@@ -377,9 +377,8 @@
                         <feature>unomi-router-karaf-feature</feature>
                         <feature>unomi-web-tracker-karaf-kar</feature>
                         <feature>unomi-groovy-actions</feature>
-                        <feature>cdp-graphql-feature</feature>
                         <feature>unomi-rest-ui</feature>
-                        <feature>unomi-graphql-ui</feature>
+                        <feature>cdp-graphql-feature-denpendencies</feature>
```

Comment:
I added it in the first try because there is some bundles in the cdp-graphql-feature-dependencies which are used by some other bundles (persistence for example). So the bundles was reloading and the unomi was not starting correctly, but with the last modifications, it's not necessary anymore. I'll do the changes